vnext/src/index.windows.js

```diff
@@ -0,0 +1,689 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ * @flow
+ */
+
+'use strict';
+
+import typeof AccessibilityInfo from './Libraries/Components/AccessibilityInfo/AccessibilityInfo';
+import typeof ActivityIndicator from './Libraries/Components/ActivityIndicator/ActivityIndicator';
+import typeof Button from './Libraries/Components/Button';
+import typeof CheckBox from './Libraries/Components/CheckBox/CheckBox';
+import typeof DatePickerIOS from './Libraries/Components/DatePicker/DatePickerIOS';
+import typeof DrawerLayoutAndroid from './Libraries/Components/DrawerAndroid/DrawerLayoutAndroid';
+import typeof FlatList from './Libraries/Lists/FlatList';
+import typeof Image from './Libraries/Image/Image';
+import typeof ImageBackground from './Libraries/Image/ImageBackground';
+import typeof InputAccessoryView from './Libraries/Components/TextInput/InputAccessoryView';
+import typeof KeyboardAvoidingView from './Libraries/Components/Keyboard/KeyboardAvoidingView';
+import typeof MaskedViewIOS from './Libraries/Components/MaskedView/MaskedViewIOS';
+import typeof Modal from './Libraries/Modal/Modal';
+import typeof Picker from './Libraries/Components/Picker/Picker';
+import typeof PickerIOS from './Libraries/Components/Picker/PickerIOS';
+import typeof Pressable from './Libraries/Components/Pressable/Pressable';
+import typeof ProgressBarAndroid from './Libraries/Components/ProgressBarAndroid/ProgressBarAndroid';
+import typeof ProgressViewIOS from './Libraries/Components/ProgressViewIOS/ProgressViewIOS';
+import typeof SafeAreaView from './Libraries/Components/SafeAreaView/SafeAreaView';
+import typeof ScrollView from './Libraries/Components/ScrollView/ScrollView';
+import typeof SectionList from './Libraries/Lists/SectionList';
+import typeof SegmentedControlIOS from './Libraries/Components/SegmentedControlIOS/SegmentedControlIOS';
+import typeof Slider from './Libraries/Components/Slider/Slider';
+import typeof Switch from './Libraries/Components/Switch/Switch';
+import typeof RefreshControl from './Libraries/Components/RefreshControl/RefreshControl';
```

Comment:
>RefreshControl [](start = 14, length = 14)

OOI, what determines the order of these imports?

Reply:
This file is almost entirely just the base react-native one.. with a few extra imports at the end of the object.

Reply:
I think the theory was Components alphabetical, then APIs alphabetical. *shrug*. I believe it is the same order as the exports